aws-api/src/main/java/com/amplifyframework/api/aws/PagedResultOperation.java

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.api.aws;
+
+import android.annotation.SuppressLint;
+
+import androidx.annotation.NonNull;
+
+import com.amplifyframework.AmplifyException;
+import com.amplifyframework.api.ApiException;
+import com.amplifyframework.api.graphql.GraphQLOperation;
+import com.amplifyframework.api.graphql.GraphQLRequest;
+import com.amplifyframework.api.graphql.GraphQLResponse;
+import com.amplifyframework.api.graphql.Page;
+import com.amplifyframework.core.Amplify;
+import com.amplifyframework.core.Consumer;
+import com.amplifyframework.logging.Logger;
+
+import java.io.IOException;
+import java.util.Objects;
+
+import okhttp3.Call;
+import okhttp3.Callback;
+import okhttp3.MediaType;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.RequestBody;
+import okhttp3.Response;
+import okhttp3.ResponseBody;
+
+/**
+ * An operation to enqueue a GraphQL request to OkHttp client,
+ * with the goal of obtaining a list of responses. For example,
+ * this is used for a LIST query vs. a GET query or most mutations.
+ * @param <T> Casted type of GraphQL result data
+ */
+public final class PagedResultOperation<T> extends GraphQLOperation<T> {
+    private static final Logger LOG = Amplify.Logging.forNamespace("amplify:aws-api");
+    private static final String CONTENT_TYPE = "application/json";
```

Comment:
I think OkHttp has a constant for this, under `MediaType`, somewhere?

Reply:
Hmm, I couldn't find a constant for it.  The [example in the OkHttp docs](https://square.github.io/okhttp/) seems to do this the same way, with a hardcoded String.  Further down in this file, it looks like this String is converted to a MediaType (`MediaType.get(CONTENT_TYPE)`), and then included on the request.